src/AWSMetadataUtilities.jl

```diff
@@ -348,17 +348,12 @@ function _generate_high_level_definition(
                 $name(args) = $service_name(\"$name\", args)
                 """
         end
-    elseif protocol == "rest-json"
-        if !isempty(required_parameters)
-            operation_definition *= "\n$name(args) = $service_name(\"$method\", \"$request_uri\", args)\n"
-        else
-            operation_definition *= """\n
-            $name() = $service_name(\"$method\", \"$request_uri\")
-            $name(args) = $service_name(\"$method\", \"$request_uri\", args)
-            """
-        end
-    elseif protocol == "rest-xml"
+    elseif protocol in ["rest-json", "rest-xml"]
```

Comment:
```suggestion
    elseif protocol in ("rest-json", "rest-xml")
```
prefer tuples for small inline collections as it's easier on the compiler

Reply:
It won't have any noticeable effect here but I thought I'd note it since it's generally good practice